package.json

```diff
@@ -4,11 +4,9 @@
   "description": "minted-frontend-boilerplate",
   "main": "index.js",
   "scripts": {
-    "build-client": "webpack --config webpack.client.js",
     "build-server": "webpack --config webpack.server.js",
     "build-app-renderer-server": "webpack --config webpack.appRendererServer.js",
-    "build": "yarn run build-client && yarn run build-server && yarn build-app-renderer-server",
-    "dev": "node build/server.js"
+    "dev": "node src/server.js"
   },
   "dependencies": {
     "@babel/polyfill": "^7.8.3",
```

Comment:
We should remove this library as we're not using it. Also, it's deprecated: https://babeljs.io/docs/en/babel-polyfill.

Reply:
good catch